docs/InterfaceKit.md

```diff
@@ -5,19 +5,19 @@ The PhidgetInterfaceKit library makes for intuitive and lightning fast developme
+|-----------|----------|-----------|
+|[connect](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams)|[__phidget__.params object](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams)|Connects the Phidget RFID|
+|[quit](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#methods)|N/A |This method requests a disconnect from the Phidget board.  The disconnected event will be dispatched when the connection has been successfully disconnected.|
 |whenReady|function|This executes a function when the Phidget InterfaceKit is ready to be used. __If you set intervals on this event, you MUST clear them on the detach event! Otherwise, you could set multiple instances of the same interval if a phidget is detached and re attached__|
-|observeOutputs|outputs|Used for asynchronously observing the digital output changes to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
-|observeInputs|inputs|Used for asynchronously observing the digital input changes to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
-|observeSensors|sensors|Used for asynchronously observing the sensor changes to the Phidget InterfaceKit board with a range of `0 - 1024`. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
-|observeRawSensors|rawSensors|Used for asynchronously observing the sensor changes to the Phidget InterfaceKit board with a range of `0 - 4096`. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeOutputs|outputs|Used for asynchronously observing the digital output changes to the Phidget InterfaceKit board.|
+|observeInputs|inputs|Used for asynchronously observing the digital input changes to the Phidget InterfaceKit board.|
+|observeSensors|sensors|Used for asynchronously observing the sensor changes to the Phidget InterfaceKit board with a range of `0 - 1024`.|
+|observeRawSensors|rawSensors|Used for asynchronously observing the sensor changes to the Phidget InterfaceKit board with a range of `0 - 4096`.|
```

Comment:
I think all of these parameters are supposed to say function since that is what they accept. A ` change handler function. ` maybe?